tests/components/modbus/test_modbus_cover.py

```diff
@@ -137,3 +141,11 @@ async def test_register_COVER(hass, regs, expected):
         scan_interval=5,
     )
     assert state == expected
+
+
+async def test_unsupported_config(hass):
+    """When user tries to run this component without discovery_info, print an error."""
+    with mock.patch.object(cover, "_LOGGER") as mock_logger:
```

Comment:
Use the `caplog` fixture instead.

https://docs.pytest.org/en/stable/logging.html#caplog-fixture

Reply:
Test updated.